src/components/dashboard/MagicLinkInfo.js

```diff
@@ -0,0 +1,116 @@
+import { CustomButton } from '../common'
+import Section from '../common/layout/Section'
+import Text from '../common/view/Text'
+import { withStyles } from '../../lib/styles'
+import illustration from '../../assets/Signup/maginLinkIllustration.svg'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import Wrapper from '../common/layout/Wrapper'
+
+Image.prefetch(illustration)
+const log = logger.child({ from: 'SignupState' })
+
+const MagicLinkInfoComponent = props => {
+  const { styles, screenProps } = props
+  const [showDialog] = useDialog()
+  const [showErrorDialog] = useErrorDialog()
+  const sendMagicEmail = () => {
+    API.sendMagicLinkByEmail(userStorage.getMagicLink())
+      .then(r => {
+        log.info('Resending magiclink')
+        fireEvent('RESENDING_MAGICLINK_SUCCESS')
+        showDialog({
+          title: 'Hocus Pocus!',
+          message: 'We sent you an email with your Magic Link',
+          onDismiss: () => screenProps.goToRoot(),
+        })
+      })
+      .catch(e => {
+        log.error('failed Resending magiclink', e.message, e)
+        showErrorDialog('Could not send magiclink email. Please try again.')
+      })
+  }
+
+  return (
+    <Wrapper backgroundColor={props.theme.colors.surface}>
+      <Section grow style={styles.section}>
+        <Section.Stack grow justifyContent="space-evenly">
+          <Section.Row alignItems="center" justifyContent="center" style={styles.row}>
+            <View style={styles.bottomContainer}>
+              <Text fontWeight="bold" fontSize={28} fontFamily="Roboto Slab" color="primary">
+                {'Abracadabra\nAnd you’re in!'}
+              </Text>
+            </View>
+          </Section.Row>
+          <Section.Row alignItems="center" justifyContent="center" style={styles.row}>
+            <View style={styles.bottomContainer}>
+              <Text fontWeight="medium" fontSize={22} fontFamily="Roboto">
+                {'By clicking your '}
+                <Text fontWeight="bold" fontSize={22} fontFamily="Roboto">
+                  {'Magic Link\n'}
+                </Text>
+                {'you can sign in from any device '}
+              </Text>
+            </View>
+          </Section.Row>
+          <Image source={illustration} style={styles.illustration} resizeMode="contain" />
+        </Section.Stack>
+      </Section>
+      <Section.Stack alignItems="stretch">
+        <CustomButton mode="outlined" dark={false} onPress={sendMagicEmail}>
+          EMAIL ME THE MAGIC LINK
+        </CustomButton>
+        <CustomButton style={styles.downBtn} onPress={screenProps.pop}>
+          OK
+        </CustomButton>
+      </Section.Stack>
+    </Wrapper>
+  )
+}
+
+const getStylesFromProps = ({ theme }) => {
+  return {
+    headerText: {
+      borderBottomWidth: 2,
+      borderBottomStyle: 'solid',
+      borderBottomColor: theme.colors.primary,
+      paddingBottom: getDesignRelativeHeight(5),
```

Comment:
use second variable=false for larger screens

Reply:
done https://github.com/GoodDollar/GoodDAPP/pull/1144/commits/513e1eaf069cb597fd1edfa1bff471b560a0703e